.changeset/sharp-crews-wave.md

```diff
@@ -0,0 +1,8 @@
+---
+"@chakra-ui/form-control": patch
+---
+
+Introduced `optionalIndicator` for `FormLabel`
```

Comment:
This is a feature, so it should be a `minor`

Reply:
We should also create a PR to the docs repo to document this feature

Reply:
updated to minor